test/e2e/E2ETest.t.sol

```diff
@@ -135,7 +151,33 @@ abstract contract E2ETest is BaseUser, Base_CType, Common {
     (uint256 _expectedCollateral,) = collateralAuctionHouse[_cType()].getCollateralBought(1, _amountToBid);
     assertEq(_expectedCollateral, COLLATERAL_AMOUNT);
 
-    _joinCoins(address(this), _amountToBid);
+    _buyCollateral(address(this), address(collateralAuctionHouse[_cType()]), 1, _expectedCollateral, _amountToBid);
+
+    uint256 _decimals = collateral[_cType()].decimals();
+    uint256 _collateralWei = _expectedCollateral / 10 ** (18 - _decimals);
+    assertEq(collateral[_cType()].balanceOf(address(this)) - _initialBalance, _collateralWei);
+
+    // NOTE: auctions(1) is deleted
```

Comment:
Do we need to create an issue to continue watching this?

Reply:
watch what specifically?